treadmill/infra/configuration.py

```diff
@@ -0,0 +1,21 @@
+from jinja2 import Template
+
+from treadmill.infra import SCRIPT_DIR
+
+
+class Configuration:
+    """Configure instances"""
+
+    def __init__(self, setup_scripts=None):
+        self.setup_scripts = setup_scripts or []
+
+    def get_userdata(self):
+        if not self.setup_scripts:
+            return ''
+
+        userdata = '#!/bin/bash -ex\n'
```

Comment:
We've added shebang to the init script now. Do we want the shebang to be pass-able as template variable?